resources/schema.gql

```diff
@@ -323,6 +327,7 @@ type DatasetFlows {
 
 type DatasetFlowsMut {
 	configs: DatasetFlowConfigsMut!
+	runs: DatasetFlowRunsMut!
```

Comment:
Since we have system flows that are not associated with some specific dataset, would it be better to make triggering a separate mutation in the root scope, just like we have for `TasksMut`?

I.e.:
```
mutation {
  tasks {
    cancelTask()
    createUpdateDatasetTask()
    createProbeTask()
  } 

  flows {
    triggerFlow()
  }
}
```
Otherwise we might end up with both.

Reply:
agreed on Slack that we will treat per-dataset and admin tasks in mutations separately